python/examples/advanced/Streaming_Data_with_Log_Rotation.ipynb

```diff
@@ -0,0 +1,332 @@
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "merged_profiles_view = None\n",
+    "for f in listdir(tmp_path):\n",
+    "    path = os.path.join(tmp_path, f)\n",
+    "    if isfile(path) and f[0] != \".\":\n",
+    "        reading_result = why.read(path)\n",
+    "        result_view =  reading_result.view()\n",
+    "\n",
+    "        if merged_profiles_view:\n",
+    "            merged_profiles_view.merge(result_view)\n",
+    "        else:\n",
+    "            merged_profiles_view = result_view\n",
+    "\n",
+    "merged_profiles_view.to_pandas()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# What's next?\n",
+    "- Check out a deeper dive into rolling logs with a flask app here. # In progress\n",
```

Comment:
Let's remove this until we actually have the flask example done?

Reply:
Deal